doc/src/pair_lj_pirani.rst

```diff
@@ -0,0 +1,163 @@
+.. index:: pair_style lj/pirani
+.. index:: pair_style lj/pirani/omp
+
+pair_style lj/pirani command
+============================
+
+Accelerator Variants: *lj/pirani/omp*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/pirani cutoff
+
+* lj/pirani = name of the pair style
+* cutoff = global cutoff (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/pirani 10.0
+   pair_coeff 1 1 4.0 7.0 6.0 3.5 0.0045
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Pair style *lj/pirani* computes pairwise interactions from an Improved
+Lennard-Jones (ILJ) potential according to :ref:`(Pirani) <Pirani>`.
+The ILJ force field is adequate to model both equilibrium and
+non-equilibrium properties of matter, in gaseous and condensed phases,
+and at gas-surface interfaces. In particular, its use improves the
+description of elementary process dynamics where the traditional
+Lennard-Jones (LJ) formulation is usually applied.
+
+
+.. math::
+
+   x = r/R_m   \\
+   n_x = \alpha*x^2 + \beta   \\
+   \gamma \equiv m  \\
+
+  V(x) = \varepsilon \cdot \left( \frac{\gamma}{ n_x - \gamma}  \left(\frac{1}{x} \right)^{n_x}
+          -  \frac{n_x}{n_x - \gamma}  \left(\frac{1}{x} \right)^{\gamma} \right) \qquad r < r_c
+
+:math:`r_c` is the cutoff.
+
+
+An additional parameter, :math:`\alpha`, has been introduced in order
+to be able to recover the traditional Lennard-Jones 12-6 with a specific
+choice of parameters. With :math:`R_m \equiv r_0 = \sigma \cdot 2^{1 / 6}`,
+:math:`\alpha = 0`, :math:`\beta = 12` and :math:`\gamma = 6`
+it is straightforward to prove that LJ 12-6 is obtained. Also, it can be
+verified that using :math:`\alpha= 4`, :math:`\beta= 8` and
+:math:`\gamma = 6`, at the equilibrium distance, the first and second
+derivatives of ILJ match those of LJ 12-6. The parameter :math:`R_m`
+corresponds to the equilibrium distance and :math:`\epsilon` to the
+well depth.
+
+
+This potential provides some advantages with respect to the standard LJ
+potential, as explained in :ref:`(Pirani) <Pirani>`: it provides a more
+realistic description of the long range behaviour and an attenuation of
+the hardness of the repulsive wall.
+
+
+This force field can be used for neutral-neutral (:math:`\gamma = 6`),
+ion-neutral (:math:`\gamma = 4`) or ion-ion systems (:math:`\gamma = 1`).
+Notice that this implementation does not include possible electrostatic
+interactions which should be eventually added by means of a hybrid style
```

Comment:
I now see that you addressed Coulombic interaction here:

Please re-word to:
does not include explicit electrostatic interactions.  If these are desired, this pair style should be used along with a Coulomb pair style (put link to pair_coul.html) and optional long-range Coulombics, which can be done using the pair_style hybrid (put link) and kspace style (put link) commands.